journalwatch.py

```diff
@@ -184,9 +184,8 @@ def parse_args():
     parser.add_argument('--priority',
                         help="Lowest priority of message to be considered.\n"
                         "A number between 7 (debug), and 0 (emergency).")
-    parser.add_argument('--loglevel',
-                        help="Level to use when sending messages to logger.\n"
-                        "Can be critical, error, warning, info or debug")
+    parser.add_argument('--verbose', '-v', action='store_true',
+                        help="Enable verbose output.\n",)
```

Comment:
Why the `\n`?

Reply:
I copy-pasted this... but now I realize the other `\n"` wasn't actually the end of a string.